changelog/no-cmpsb.dd

```diff
@@ -0,0 +1,24 @@
+# `memcmp()` compares are no longer performed for struct equality tests, memberwise comparisons are done instead, per the spec
```

Comment:
This will add a literal `#` at the beginning of the line, see: http://dtest.dlang.io/artifact/website-6d079ba9822f488660df930751bffd99a564cbfd-9b21681f7b99def3027cda434bb08739/web/changelog/pending.html#no-cmpsb.

Reply:
I thought Ddoc did markdown now.

Reply:
It's still not activated by default ;-)